Bio/Phylo/PhyloXML.py

```diff
@@ -711,21 +711,13 @@ def __rmul__(self, other):
         """Conduct reverse multiplication between values of two Confidence objects."""
         return other * self.value
 
-    def __div__(self, other):
```

Comment:
@etal I'd particularly like your eyes on this change, the original was introduced in cd577805c82a28fb40cdef21c78d0d29d2ea7fda

Reply:
Moved to #2550